test/integration/framework/etcd.go

```diff
@@ -198,7 +198,7 @@ func EtcdMain(tests func() int) {
 		// like k8s.io/klog/v2.(*loggingT).flushDaemon()
 		// TODO(#108483): Reduce this number once we address the
 		//   couple remaining issues.
-		if dg := runtime.NumGoroutine() - before; dg <= 10 {
+		if dg := runtime.NumGoroutine() - before; dg <= 15 {
```

Comment:
`test/integration/controlplane/transformation` integration tests leak goroutines because the kms transformer (grpc goroutines) are being created but never closed.

https://github.com/kubernetes/kubernetes/blob/18ce801beac34489f313d6b1223a1a5f07200cb6/staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go#L245

This is work we need to do to prevent this leak in our integration tests for `test/integration/controlplane/transformation`. I didn't want to make any of those improvements in this PR to prevent scope creep.

Reply:
can you file a follow up issue to shut down the goroutines and lower this back?

Reply:
opened https://github.com/kubernetes/kubernetes/issues/111674 to track this

Reply:
what is the ETA to fix this? during this test freeze?

it took wojtek a lot of time to get down to this number it will be sad now we start bumping it again

Reply:
agreed, please prioritize https://github.com/kubernetes/kubernetes/issues/111674, ideally before test freeze